packages/angular/src/providers/toast/toast.service.ts

```diff
@@ -8,56 +8,29 @@
  */
 
 import { Injectable } from '@angular/core';
+import { ShowToastResult } from '@siemens/ix';
 import {
-  getToastContainer,
-  toast,
-  ToastConfig as IxToastConfig,
-} from '@siemens/ix';
-import { ToastConfig } from '@siemens/ix-angular/common';
+  ToastService as BaseToastService,
+  ToastConfig,
+} from '@siemens/ix-angular/common';
 
 @Injectable({
   providedIn: 'root',
 })
-export class ToastService {
-  setPosition(position: 'bottom-right' | 'top-right') {
-    getToastContainer().position = position;
+export class ToastService extends BaseToastService {
+  constructor() {
+    super();
   }
 
-  getPosition() {
-    return getToastContainer().position;
+  public getPosition(): 'bottom-right' | 'top-right' {
```

Comment:
```suggestion
  public getPosition(): ToastPosition {
```

There is already a ToastPosition typ